### PR TITLE
DIT-777 case notes display changes

### PR DESCRIPTION
**Pull request by retroredge**

### CODE REVIEW

File: test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/TextWithLineFeedsViewSpec.scala [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+package uk.gov.hmrc.tariffclassificationfrontend.views.partials
+
+import uk.gov.hmrc.tariffclassificationfrontend.views.ViewMatchers._
+import uk.gov.hmrc.tariffclassificationfrontend.views.ViewSpec
+import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.text_with_line_feeds
+
+class TextWithLineFeedsViewSpec extends ViewSpec {
+
+  "Progressing Disclosure" should {
```
Comment (ed0906) [inline test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/TextWithLineFeedsViewSpec.scala]: Think this should be "Text with Line Feeds"

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html [hunk 2]
```diff
@@ -40,10 +40,9 @@
                     event.details.`type` match {
                         case EventType.NOTE => {
                             val body = event.details.asInstanceOf[Note]
-                            progressive_disclosure(
+                            text_with_line_feeds(
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html]: Can we call this something more intuitive, e.g. `paragraph`? or `text_block`?

File: test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/TextWithLineFeedsViewSpec.scala (hunk 1 above)
Comment (retroredge) [reply test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/TextWithLineFeedsViewSpec.scala]: Good spot. I've actually removed this partial and the test now and have used Steve's CSS idea to preserve the linefeeds

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html (hunk 2 above)
Comment (retroredge) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details_events_table.scala.html]: it's gone :)

**Pull request merged**